core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -18,18 +18,55 @@
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
     </script>
+
   {% endif %}
 
   {{dependencies_html}}
+  <style>
+   @media(max-width: 768px) {
+     /*To prevent Navbar From Collapsing On Exploration Player, so that title can appear*/
```

Comment:
```
    /* This prevents the navbar from collapsing in the exploration page,
    so that the exploration title can be shown. */
```

(Pay attention to spacing, case, and punctuation.)